cli/diff.rs

```diff
@@ -3,6 +3,142 @@
 use crate::colors;
 use dissimilar::{diff as difference, Chunk};
 
+/// Print diff of the same file_path, before and after formatting.
+///
+/// Diff format is loosely based on Github diff formatting.
+pub fn diff(orig_text: &str, edit_text: &str) -> String {
```

Comment:
Sorry, the diff here wasn't going to be pretty no matter what, so I took the liberty of reorganizing the file. For the most part it's largely the same other than a few fixes. I've commented everywhere that's changed.